nixos/modules/programs/firejail.nix

```diff
@@ -20,13 +19,37 @@ let
     '';
   };
 
+  wrappedPkgs = map (pkg:
+    pkgs.symlinkJoin {
+      name = "firejail-" + pkg.name;
+      paths = [ pkg ];
+      buildInputs = with pkgs; [ tree ];
```

Comment:
I don't think this should be needed as a build input. All of the below commands are included in stdenv.